Techome/Techome/Views/Trend/TrendView.swift

```diff
@@ -0,0 +1,221 @@
+//
+//  TrendView.swift
+//  Techome
+//
+//  Created by 한택환 on 2022/06/09.
+//
+
+import SwiftUI
+
+struct TrendLayout {
+    
+    struct Paddings {
+        //TrendView Paddings
+        static let dayRecordPadding: CGFloat = 8
+        static let caffeineRecordRowVerticalPadding: CGFloat = 15
+        static let caffeineRecordRowHorizontalPadding: CGFloat = 20
+        static let chartPadding: CGFloat = 8
+        static let textVerticalPadding: CGFloat = 5
+        static let averageCaffeineWeekPadding: CGFloat = 7
+        static let sideEffectRecordCellHorizontalPadding: CGFloat = 21
+        static let averageCaffeineAmountPadding: CGFloat = 15
+        static let caffeineRecordAmountUnitPadding: CGFloat = 1
+        static let TotalListViewPadding: CGFloat = 20
+    }
+    
+    //TrendView Sizes
```

Comment:
```suggestion
    /// TrendView Sizes
```

///를 사용하면 자동완성에서도 주석이 나타난답니다 ✨

Reply:
///는 몰랐어요!  감사합니다!!